package.json

```diff
@@ -55,9 +65,13 @@
   "devDependencies": {
     "@types/mocha": "^10.0.9",
     "@types/node": "^22.9.0",
+    "@types/vscode": "^1.62.0",
```

Comment:
Why's this change downgrading the package version?

Reply:
This version and the engines > vscode version should be kept in sync, and thought it'd be better to er on the side of caution (supporting more versions than less). I can bump them both back to 1.93 or 1.95

Reply:
Makes sense 👍